src/JuMP.jl

```diff
@@ -100,6 +100,9 @@ mutable struct Model <: AbstractModel
     # obj#::QuadExpr
     # objSense::Symbol
 
+    # We use the constraint reference value to have a concrete type
+    constrainttosolverconstraint::Dict{UInt64,UInt64}
```

Comment:
1) I don't like how this is untyped.
2) I think it's incorrect, because constraint reference values are allowed to overlap if they're of different types.

Reply:
I knew you were not going to like it :D